### DISCUSSION TITLE
Ignore hooks

### DISCUSSION THREAD

**Discussion by bdvorianov** [Escape-Technologies/mookme#76] [Q&A] [answered]
Hello folks! 

I'm newbie to mookme and few days ago I set up some hooks for my project, pretty basic stuff like linter, build on push and so on. I'm curious if there a mookme parameter or something I can do to skip the hooks. 

Thanks.

**Comment by LMaxence** [comment, marked as answer]
Hello ! 

First of all, many thanks for giving Mookme a try :) It depends on how you invoked Mookme in the first place:

- If mookme is triggered as part of your usual git hooks (meaning as a pre-commit or pre-push git hook in your case), you can avoid it being executed using the `--no-verify` option.

```
# no hooks will be executed
git commit -m "something" --no-verify
```

- If you invoke directly with the CLI (like in a CI or something like this), you will have to include some logic in your script to skip the command in any logical case you don't want it to run

- If neither options are suitable, you can try to use the `onlyOn` [step option](https://mookme.org/references/#hook-files)

Hope I could help ! Feel free to tell me if it didn't help

**Reply by bdvorianov** [reply]
Thanks! 🙌🏻

I'll check it out, seems like `--no-verify` should do for my case.

BTW, Mookme feels good so far. It does what I need without additional pain. Thank you. 🤝

**Reply by bdvorianov** [reply]
Yep, it worked for pre-commit, so I suppose it will do for other. ✅
